packages/twenty-front/src/modules/object-record/record-table/record-table-row/components/RecordTableActionRow.tsx

```diff
@@ -36,33 +36,39 @@ const StyledEmptyTd = styled.td`
 `;
 
 type RecordTableActionRowProps = {
+  draggableId: string;
+  draggableIndex: number;
   LeftIcon: IconComponent;
   text: string;
   onClick?: (event?: React.MouseEvent<HTMLTableRowElement>) => void;
 };
 
 export const RecordTableActionRow = ({
+  draggableId,
+  draggableIndex,
   LeftIcon,
   text,
   onClick,
 }: RecordTableActionRowProps) => {
   const theme = useTheme();
 
-  const visibleColumns = useRecoilComponentValueV2(
-    visibleTableColumnsComponentSelector,
-  );
+  const { visibleTableColumns } = useRecordTableContextOrThrow();
 
   return (
-    <StyledTrContainer onClick={onClick}>
+    <StyledRecordTableDraggableTr
```

Comment:
why do we need to make the RecordTableActionRow draggable?

Reply:
If this one is not draggable we can't drag and drop properly rows within an empty group, the load more button and create new will not move, so it looks like it's not working